3.4/administration.md

```diff
@@ -46,6 +46,7 @@ Other Topics
 
 - [Configuration](administration-configuration.html)
 - [Backup & Restore](backup-restore.html)
+- [Hot backup](hot-backup.html)
```

Comment:
There is no file `3.4/hot-backup.html`!

Would fit better as a section in Backup & Restore, but with the sub-pages this would get quite nested. WDYT @sleto-it ?